### DISCUSSION TITLE
Problems loading performance

### DISCUSSION THREAD

**Discussion by Maximo68** [probonopd/MiniDexed#561] [Q&A] [answered]
HI! My name is Maximo68 and I'm new to the group. The performances seem to work fine but when I load the performance pseudosilk the system freezes when it no longer receives commands. I have a Raspberry Pi4b.

**Comment by probonopd** [comment]
@Banana71 do you have an idea what might be causing this?

@Maximo68 Have you tried another SD card?

**Reply by Banana71** [reply]
Which version of miniDexed are you using?

**Reply by Maximo68** [reply]
The card in use is a Samsung EVO Plus (1) 32GB. I'll try changing tabs... The minidexed version was downloaded from this address: https://drive.google.com/file/d/1ol9fCtepMS7_lNUFYss7EbOfytvl94Uj/view. Among other things, it is the only one that works. I tried the latest MiniDexed_2023-09-23 but the system doesn't boot....

**Reply by probonopd** [reply]
>  I tried the latest MiniDexed_2023-09-23 but the system doesn't boot....

Wow, that'd be a grave bug. Were you using the same configuration?

Where is the version on Google Drive even coming from? We certainly can't support unknown "random" builds here.

**Reply by Banana71** [reply]
Your version uses a 3.3V DAC. Do you also have a 3.3V DAc or the 5V version as described here?
Actually the wiring is the same except for the supply from the DAC.
Are the performances current?

**Reply by Maximo68** [reply]
It was downloaded from a YouTube video that reported the Minidexed project. The clue is this: https://www.youtube.com/watch?v=QYkz6rz20Dk

**Reply by Banana71** [reply]
We should find out why the version from 2023/09/23 doesn't boot for you.

**Reply by Maximo68** [reply]
My version uses a 5V PCM5102A I2S DAC

**Reply by Banana71** [reply]
> My version uses a 5V PCM5102A I2S DAC

And is your DAC also connected to the 5Volt PIN?  or to 3.3 volts?

**Comment by Maximo68** [comment]
My PCM5102A I2S DAC was incorrectly connected to the 3.3V output. I made the connection to the 5V. Thanks for the tip. However minidexed always crashes on pseudosilk performance. In reality it plays it perfectly but as soon as I move the encoder to the next or previous performance it stops.

**Reply by Maximo68** [reply, marked as answer]
Hi, everything seems to be working now. I made a change to the minidexed.ini file regarding the LCD module which was not set correctly. Thank you for your support.

**Reply by probonopd** [reply]
Thanks @Maximo68 for asking and thanks @Banana71 for helping! Enjoy your instrument.
